prov/gni/include/gnix_ep.h

```diff
@@ -223,12 +250,57 @@ __msg_match_fab_req(struct dlist_entry *item, const void *arg)
 				(GNIX_ADDR_EQUAL(req->addr, *addr_ptr)));
 }
 
+/*
+ * EP related internal helper functions
+ */
```

Comment:
Double underscore is for file internal function.  These should be renamed (we agreed to use `_gnix` prefix for gni provider internal functions).